maths/path_length.py

```diff
@@ -10,31 +10,31 @@
 """
 
 
-def discrete_path_length(seq: list[tuple[float, float]]) -> float:
+def discrete_path_length(cordinateSequence: list[tuple[float, float]]) -> float:
```

Comment:
Python variable names please.  See CONTRIBUTING.md.
```suggestion
def discrete_path_length(cordinate_sequence: list[tuple[float, float]]) -> float:
```